src/nmodl/deriv.cpp

```diff
@@ -168,20 +173,21 @@ void solv_diffeq(Item* qsol,
         vectorize_substitute(qsol, buf);
     } else { /* kinetic */
         if (vectorize) {
-            Sprintf(
-                buf,
-                "%s%s_thread(&(_thread[_spth%d].literal_value<void*>()), %d, _slist%d, _dlist%d, _p, &%s, %s, %s\
-, _linmat%d, _p, _ppvar, _thread, _nt);\n",
-                ssprefix,
-                method->name,
-                listnum,
-                numeqn,
-                listnum,
-                listnum,
-                indepsym->name,
-                dindepname,
-                fun->name,
-                listnum);
+            // sparse_thread, _ss_sparse_thread
+            Sprintf(buf,
+                    "%s%s_thread(&(_thread[_spth%d].literal_value<void*>()), %d, "
```

Comment:
`note to myself`: understand/review the `literal_value()` usage.

Reply:
understand/review/despair/condemn 😄